app/src/main/java/com/io/movio/ui/movielist/adapter/MoviesAdapter.kt

```diff
@@ -33,12 +41,26 @@ class MoviesAdapter(private val listener: ItemOnClickListener)
         notifyDataSetChanged()
     }
 
-    override fun getItemCount(): Int  =  movies.size
+    override fun getItemCount(): Int = movies.size
 
     class MovieViewHolder(val binding: MovieRowBinding) : RecyclerView.ViewHolder(binding.root)
 
     interface ItemOnClickListener {
         fun onItemClick(id: Int)
     }
 
+}
+private fun getGenreToString(genre: List<Int>): String {
+    var genres = ""
+    for (i in genre) {
+        val genreCapital = Genre.getByValue(i).toString()
+        val genreString = nameFormatter(genreCapital)
+        genres = genres.plus(genreString.plus(", "))
+    }
+    return genres.dropLast(2)
+}
+
+private fun nameFormatter(genreCapital: String): String {
```

Comment:
add the genres as string resources. Then introduce them as a property in the enums

enum Genre(@StringRes  val name: Int, val id: Int)